libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Teams/Actions/GetMeetingParticipant.cs

```diff
@@ -0,0 +1,148 @@
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive.Actions
+{
+    /// <summary>
+    /// Calls TeamsInfo.GetMeetingParticipantAsync and sets the result to a memory property.
+    /// </summary>
+    public class GetMeetingParticipant : Dialog
+    {
+        /// <summary>
+        /// Class identifier.
+        /// </summary>
+        [JsonProperty("$kind")]
+        public const string Kind = "Teams.GetMeetingParticipant";
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="GetMeetingParticipant"/> class.
+        /// </summary>
+        /// <param name="callerPath">Optional, source file full path.</param>
+        /// <param name="callerLine">Optional, line number in source file.</param>
+        [JsonConstructor]
+        public GetMeetingParticipant([CallerFilePath] string callerPath = "", [CallerLineNumber] int callerLine = 0)
+            : base()
+        {
+            this.RegisterSourceLocation(callerPath, callerLine);
+        }
+
+        /// <summary>
+        /// Gets or sets an optional expression which if is true will disable this action.
+        /// </summary>
+        /// <example>
+        /// "user.age > 18".
+        /// </example>
+        /// <value>
+        /// A boolean expression. 
+        /// </value>
+        [JsonProperty("disabled")]
+        public BoolExpression Disabled { get; set; } 
+
+        /// <summary>
+        /// Gets or sets property path to put the value in.
+        /// </summary>
+        /// <value>
+        /// Property path to put the value in.
+        /// </value>
+        [JsonProperty("property")]
+        public StringExpression Property { get; set; }
+
+        /// <summary>
+        /// Gets or sets the expression to get the value to use for meeting id.
+        /// </summary>
+        /// <value>
+        /// The expression to get the value to use for meeting id. If this is missing, then the current turn Activity.TeamsChannelData.Meeting.Id will be used.
+        /// </value>
+        [JsonProperty("meetingId")]
+        public StringExpression MeetingId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the expression to get the value to use for participant id.
+        /// </summary>
+        /// <value>
+        /// The expression to get the value to use for participant id. If this is missing, then the current turn Activity.From.AadObjectId will be used.
+        /// </value>
+        [JsonProperty("participantId")]
+        public StringExpression ParticipantId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the expression to get the value to use for tenant id.
+        /// </summary>
+        /// <value>
+        /// The expression to get the value to use for tenant id. If this is missing, then the current turn Activity.TeamsChannelData.Tenant.Id will be used.
+        /// </value>
+        [JsonProperty("tenantId")]
+        public StringExpression TenantId { get; set; }
```

Comment:
>public StringExpression TenantId { get; set; } [](start = 8, length = 46)

Another way you can set the default value is like this:
```
      public StringExpression TenantId {get;set;}  = "=turn.activity.channelData.meetingInfo.Id";
```
If you put that into the .schema file then composer would show that path in the ui.